### PR TITLE
Create MSK Serverless scrape config

### PR DESCRIPTION
**Pull request by brituck**
InfraAPI is using AWS MSK Serverless for event streaming. Adds MSK serverless metrics config for atlas. See https://docs.aws.amazon.com/msk/latest/developerguide/serverless-monitoring.html for metrics details.

### CODE REVIEW

File: atlas-cloudwatch/src/main/resources/msk-serverless.conf [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  cloudwatch {
+
+    // https://docs.aws.amazon.com/msk/latest/developerguide/serverless-monitoring.html
+    msk-serverless-cluster = {
+      namespace = "AWS/Kafka"
+      period = 1m
+      end-period-offset = 5
+
+      dimensions = [
+        "Cluster Name",
```
Comment (brituck) [inline atlas-cloudwatch/src/main/resources/msk-serverless.conf:11]: Is this correct with the space? That's how it's listed in AWS, but not sure if we do some sort of whitespace removal

File: atlas-cloudwatch/src/main/resources/msk-serverless.conf [hunk 2]
```diff
@@ -0,0 +1,104 @@
+atlas {
+  cloudwatch {
+
+    // https://docs.aws.amazon.com/msk/latest/developerguide/serverless-monitoring.html
+    msk-serverless-cluster = {
+      namespace = "AWS/Kafka"
+      period = 1m
+      end-period-offset = 5
```
Comment (brituck) [inline atlas-cloudwatch/src/main/resources/msk-serverless.conf:8]: MSK publishes these metrics once per minute. Do we need an offset? Was following other per-minute metrics configs like Dynamo

Comment (manolama) [reply atlas-cloudwatch/src/main/resources/msk-serverless.conf:8]: I don't think we use it now but leave it in and we'll clean it up later.

File: atlas-cloudwatch/src/main/resources/msk-serverless.conf [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+    msk-serverless-consumergroup = {
+      namespace = "AWS/Kafka"
+      period = 1m
+      end-period-offset = 5
+
+      dimensions = [
+        "Cluster Name",
+        "Topic",
+        "Consumer Group"
```
[the remark this answers is not in the document]
Comment (manolama) [reply atlas-cloudwatch/src/main/resources/msk-serverless.conf:82]: Also, add these three to the tag map in https://github.com/Netflix-Skunkworks/iep-apps/blob/main/atlas-cloudwatch/src/main/resources/reference.conf#L236 with dotted notation for Atlas please.

**Review by manolama — APPROVED** [review]

**Pull request merged**